wiki/en/en-Running-a-Server.md

```diff
@@ -10,7 +10,7 @@ permalink: "/wiki/Running-a-Server"
 ## Do I need to run a server to use Jamulus?
 
 NO.
-{: .doubletextsize .red }
+ {: .doubletextsize .red }
```

Comment:
Did you check if this works locally too?

Reply:
Yes. It works.

Reply:
Ok. Thanks for the clarification here.